### PR TITLE
Home Screen Filter Tags

### PR DESCRIPTION
**Pull request by Aditya13s**

### CODE REVIEW

File: app/src/main/java/com/doubtless/doubtless/screens/doubt/HomeMainScreenFragment.kt [hunk 1]
```diff
@@ -38,31 +39,39 @@ class HomeMainScreenFragment : Fragment() {
         if (_navigator != null) navigator = _navigator
 
         viewModel = getViewModel()
-        viewModel.fetchTags()
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        viewModel.fetchTags()
```
Comment (sidsharma2002) [inline app/src/main/java/com/doubtless/doubtless/screens/doubt/HomeMainScreenFragment.kt]: if there is already data fetched then dont fetch again, if (viewmodel.result.data != null) something like this

**Review by sidsharma2002 — CHANGES_REQUESTED** [review]

**Comment by Aditya13s** [comment]
@sidsharma2002 check the changes

**Comment by sidsharma2002** [comment]
@Aditya13s add analytics

**Comment by Aditya13s** [comment]
@sidsharma2002 please check the pull request

**Pull request merged**